com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -63,12 +73,34 @@ public static ActionSpec MakeDiscrete(params int[] branchSizes)
             return actuatorSpace;
         }
 
+        /// <summary>
+        /// Set action size fields related to continuous actions.
+        /// </summary>
+        /// <param name="numContinuousActions">Number of Continuous Actions</param>
+        public void SetContinuous(int numContinuousActions)
+        {
+            m_NumContinuousActions = numContinuousActions;
+        }
+
+        /// <summary>
+        /// Set action size fields related to discrete actions.
+        /// </summary>
+        /// <param name="numDiscreteActions">Number of Discrete Actions</param>
+        /// <param name="branchSizes">The array of branch sizes for the discrete action space.  Each index
+        /// contains the number of actions available for that branch.</param>
+        public void SetDiscrete(int numDiscreteActions, int[] branchSizes)
```

Comment:
Can SetDiscrete only take a single branchSizes argument ?

Reply:
not sure I understand this question.

Reply:
I think he means `public void SetDiscrete(int[] branchSizes)`. 
This makes sense to me; only `m_NumContinuousActions` and `m_DiscreteBranchSizes` is necessary in ActionSpec for all information.